terratag_test.go

```diff
@@ -84,6 +84,28 @@ func TestTerraform1o2WithFilter(t *testing.T) {
 	testTerraformWithFilter(t, "15_1.2_filter", "azurerm_resource_group|aws_s3_bucket")
 }
 
+func TestTerragruntWithCache(t *testing.T) {
```

Comment:
The test is still work in progress. but can already be executed at works well.
Need to add the verification to "Expected" code.